### ISSUE TITLE
Improve syntactic highlight

### ISSUE THREAD

**Issue by magnus-madsen** [flix/vscode-flix#271]
I would like this to be colored better:

![image](https://github.com/flix/vscode-flix/assets/2287953/0bc94e19-a04c-4a3a-a359-f8ac5758f2e4)

(Note this is not a valid Flix program, but rather part of the IR).

In any case, I am hoping a simple change to the `flix.tmLanguage.json` file is enough.

**Comment by magnus-madsen** [comment]
@JonathanStarup CC

**Comment by magnus-madsen** [comment]
We also need for "untag".

**Comment by sockmaster27** [comment]
Do I understand it correctly that the problem is that the identifiers are getting split up by the dollar sign?
Because in that case, it doesn't seem to involve `flix.tmLanguage.json` - or at least I can't find where it would.

**Comment by magnus-madsen** [comment]
> Do I understand it correctly that the problem is that the identifiers are getting split up by the dollar sign? Because in that case, it doesn't seem to involve `flix.tmLanguage.json` - or at least I can't find where it would.

Yes, I was thinking `getErr$12345` would be in one color, if possible.

**Comment by sockmaster27** [comment]
How are you viewing the file without having errors disabling the highlighting?

**Comment by magnus-madsen** [comment]
This is not a real file. Rather it is when you right click and choose "Show AST" and select one of the phases.
